src/Smidge/Models/RequestModel.cs

```diff
@@ -17,7 +17,7 @@ protected RequestModel(string valueName, IUrlManager urlManager, IActionContextA
             if (requestHelper is null)throw new ArgumentNullException(nameof(requestHelper));
 
             //default 
-            LastFileWriteTime = DateTime.Now;
+            LastFileWriteTime = DateTime.MinValue;
```

Comment:
I thought I asked about this before but can't find the convo. What is this change addressing?

Reply:
This default is updated here to the right one: https://github.com/Shazwazza/Smidge/blob/b0383c3b7e2b667402451dcff7055209ceabd428/src/Smidge/Controllers/CompositeFileCacheFilterAttribute.cs#L36
DateTime.MinValue doesn't do anything whereas DateTime.Now retrieves the date/time and timezone information etc.